modules/ROOT/pages/manage-index.adoc

```diff
@@ -1,13 +1,34 @@
 = Manage Runtime Fabric
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
 
-After installing and configuring Runtime Fabric and deploying Mule applications and API proxies, there are additional tasks you may need to perform.
+After you install and configure Runtime Fabric and deploy Mule applications and API proxies, there are additional tasks you might need to perform.
```

Comment:
first use in primary topic:
```suggestion
After you install and configure Anypoint Runtime Fabric and deploy Mule applications and API proxies, you might need to perform additional tasks.
```
...tasks, depending on...?
Also, GLOBALLY avoid using syntax that uses "there is" or "there are" as pseudo-subjects

Reply:
Other suggestion:
```suggestion
After you install and configure Runtime Fabric and deploy Mule applications and API proxies, you can then view, monitor, update, and maintain your environment, and manually uninstall Runtime Fabric if necessary.
```

Reply:
Minor suggestion to not highlighting uninstall as it's not part of the typical lifecycle.